packages/host/public/index.html

```diff
@@ -1,8 +1,10 @@
 <html>
   <head>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
     <link
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&display=swap"
```

Comment:
Wouldn't these not work if you're on a lan?

We're using the `fontsource` package for `Inter` for self-hosting the font and we have all the different weights in node modules.. maybe they're just not getting loaded properly?

![image](https://user-images.githubusercontent.com/35858975/146845579-9cdedd3b-34cd-418a-9cd6-193bfc2c1e0e.png)

Reply:
huh. I didn't realise we were using `fontsource` oops. removing the `link` from the html does screw up the fonts though, so if there is fontsource it isn't used for the web version..??